frontend/pages/hubs/[hubUrl].tsx

```diff
@@ -32,6 +32,8 @@ import AddIcon from "@mui/icons-material/Add";
 import { Theme } from "@mui/material/styles";
 import theme from "../../src/themes/theme";
 import BrowseContext from "../../src/components/context/BrowseContext";
+import { ThemeProvider } from "@mui/material/styles";
+import { transformThemeData } from "../../src/themes/transformThemeData";
```

Comment:
Implemented a function to transform theme data received from the API into a structured theme object for use in the frontend.